content/en/docs/helm/helm_dependency.md

```diff
@@ -55,6 +55,9 @@ If the dependency chart is retrieved locally, it is not required to have the
 repository added to helm by "helm add repo". Version matching is also supported
 for this case.
 
+### Best Practices
```

Comment:
This file is auto-generated from the Helm CLI's help text. See https://github.com/helm/helm-www#updating-the-helm-cli-reference-docs for more information.

Can you please remove this?

Reply:
Hey @bacongobbler, apologies and thanks for the correction. Done.